examples/auction/auction_cc.cpp

```diff
@@ -21,25 +22,96 @@
 #define AUCTION_ALREADY_CLOSED "AUCTION_ALREADY_CLOSED"
 #define AUCTION_STILL_OPEN "AUCTION_STILL_OPEN"
 
-static std::string SEP = ".";
-static std::string PREFIX = SEP + "somePrefix" + SEP;
+#define INITALIZED_KEY "__initalized"
```

Comment:
Isn't it ```#define INITIALIZED_KEY "__initialized"```

Reply:
definitely a typo, but being a constant at least not leading to bugs :-) (i've fixed it now, though)